include/nanobind/eigen/sparse.h

```diff
@@ -0,0 +1,112 @@
+/*
+    nanobind/eigen/sparse.h: type casters for sparse Eigen
+    vectors and matrices
```

Comment:
We do not support sparse vectors though, right? I remember there being a pybind11 bug report about it.